packages/packages/editor/src/init.tsx

```diff
@@ -1,6 +1,9 @@
 import ReactDOM from 'react-dom';
 import Shell from './components/shell';
+import { StoreProvider, createStore } from '@elementor/store';
 
 export default function init( domElement: HTMLElement ): void {
-	ReactDOM.render( <Shell />, domElement );
+	const store = createStore();
+
+	ReactDOM.render( <StoreProvider store={ store }><Shell /></StoreProvider>, domElement );
```

Comment:
I think it will be weird, we need to create the store at this specific point, where should we create the store instance? inside the StoreProvier? it's just an export of Redux Provider..

Reply:
Seems good to me, as it is now.